test/view/repeat_view.cpp

```diff
@@ -97,6 +98,13 @@ int main() {
 				const std::vector<int>*>);
 		CHECK(first->size() == N);
 	}
+	{
+		auto v = ranges::view::ext::repeat(9) | ranges::view::take(10);
+		static_assert(ranges::View<decltype(v)>);
+		static_assert(ranges::RandomAccessIterator<decltype(v.begin())>);
+		// static_assert(ranges::SizedView<decltype(v)>);
```

Comment:
Ditto `SizedView` => `SizedRange`?

Reply:
I've been thinking about what `repeat_view::size` should return. Even if we bring the `cardinality` type over from range-v3, it feels like an arbitrary result (IIUC the size is -3).

Reply:
`repeat_view` should have no member `size` - it's not finite, so it has no well-defined size.  (And yes, it didn't occur to me that we don't have range-v3's cardinality machinery so `take_view<repeat_view<Foo>>` will necessarily not be a sized range.)

range-v3's `cardinality` isn't strictly a size: values `>= 0` are sizes, values `< 0`:
* `unknown` is potentially infinite, (e.g., a `filter_view` adapting an `infinite` underlying range)
* `finite` is not known statically but definitely finite (e.g. `vector`)
* `infinite` is definitely infinite. (e.g. `repeat_view`)

Reply:
In other words: ignore this suggestion.